bin/sandbox_rails

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+
+warn %{
+DEPRECATION: bin/sandbox_rails has been replaced by bin/rails-sandbox, please
+use that command instead.
+}.strip
+
+exec "#{__dir__}/rails-engine"
```

Comment:
This is an error in solidus_dev_support (that I think I introduced myseld 😓), it should point to `bin/rails-sandbox`.